src/plugins/handlers.js

```diff
@@ -260,6 +265,10 @@ function handlers(app, opts, done){
     cm.execCommand('replace');
   }
 
+  function goToEnd(){
+    cm.execCommand('goDocEnd');
```

Comment:
I think this should be built into the `documents.create` method.  I don't think there is ever a scenario where you would want to create the document but not focus the end